src/apps/backend/modules/account/types.py

```diff
@@ -68,3 +68,9 @@ class AccountErrorCode:
     USERNAME_ALREADY_EXISTS: str = "ACCOUNT_ERR_01"
     BAD_REQUEST: str = "ACCOUNT_ERR_04"
     PHONE_NUMBER_ALREADY_EXISTS: str = "ACCOUNT_ERR_05"
+
+
+@dataclass(frozen=True)
+class UpdateAccountProfileParams:
```

Comment:
From your implementation in view, its not clear if both first_name and last_name are required by client to sent. If not sent, you are default to empty string and then setting it in database which is not desirable.